frontend-too/src/components/pages/Home.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { Form, Button } from 'react-bootstrap';
+import { useNavigate } from 'react-router-dom';
+
+const Home = () => {
+  let navigate = useNavigate();
+  let name = localStorage.getItem('name');
+  useEffect(() => {
```

Comment:
nitpicky formatting but drop a new line between your variable declarations and the useEffect